style.css

```diff
@@ -0,0 +1,83 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: content-box;
+}
+
+#add-prod-div {
+    margin: 10vh auto;
+    width: 450px
+}
+
+#add-prod-div h2 {
+    margin: 6px auto;
+    text-align: center;
+    color: #0ba589;
+    font-family: 'Poppins', sans-serif;
+}
+
+#add-prod-form {
+    display: grid;
+    grid-template-columns: 100px 280px;
+}
+
+#add-prod-form input:focus,
+#add-prod-form textarea:focus {
+    border: 1px solid #196b5c !important;
```

Comment:
If you can see in our design, we don't have borders for any field in this page, so I suggest you remove them, it will look prettier.